.github/workflows/build-and-test.yml

```diff
@@ -39,19 +39,23 @@ jobs:
         sudo apt-get install -y --no-install-recommends fftw3-dev
```

Comment:
add `catch2`?

Reply:
Doesn't exist in ubuntu 20.04: https://packages.ubuntu.com/focal/catch2

Reply:
Oh, looks like it's only in Ubuntu 20.10: https://packages.ubuntu.com/search?keywords=catch2&searchon=names&suite=groovy&section=all

Reply:
Anyway, the auto-fallback to GitHub works pretty good: https://github.com/mixxxdj/libKeyFinder/pull/3/files#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0dR24-R33

Reply:
Please update README.md to note this dependency and that it will be automatically downloaded when building tests if not already installed. If it is not installed and libKeyFinder is built offline, then libKeyFinder can be built by disabling tests.